CMakeLists.txt

```diff
@@ -360,8 +360,12 @@ if(BUILD_PIPS_IPM OR BUILD_PIPS_NLP)
     # - scalapack and blacs should be set in ${SCALAPACK_LIBRARIES} using Toolchain.cmake (or hardcoded here)
     # - mpifort also needed
     # - parmetis 5.0 or later required by mumps but no need to specify it here: it should be detected by the build system at this point
-    set(MUMPS_LIBRARY "-L${MUMPS_DIR}/lib -ldmumps -lmumps_common -lpord -lscalapack-openmpi -lblacs-openmpi  -lblacsF77init-openmpi -lblacsCinit-openmpi -lmpi_mpifh")
+    if(NOT SCALAPACK_LIBRARIES)
+      set(SCALAPACK_LIBRARIES "-lscalapack-openmpi -lblacs-openmpi  -lblacsF77init-openmpi -lblacsCinit-openmpi") 
+    endif(NOT SCALAPACK_LIBRARIES)
+    set(MUMPS_LIBRARY "-L${MUMPS_DIR}/lib -ldmumps -lmumps_common -lpord -lmpi_mpifh ${SCALAPACK_LIBRARIES}")
```

Comment:
-lmpi_mpifh is openmpi specific. How about we remove it from this 'set' and ask the user specify this mpi fortran library in SCALAPACK_LIBRARIES or hardcode it. And update the instructions above to " - mpifortran also needs to be specified, either in the Toolchain.cmake (e.g., append it to 'SCALACK_LIBRARIES') or hardcoded below"

Reply:
Alright. Just keep in mind that the MPI Fortran dependency -lmpi_mpifh comes from MUMPS not from scalapack. I believe scalapack uses only the C interface of MPI.  So I'll create a MUMPS_LIBRARIES variable I guess.

Reply:
ah, ok. better that way indeed

Reply:
@michel2323 let me know when you put in MUMPS_LIBRARY so I can merge the PR. thanks!